fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -114,6 +117,8 @@
 
     private final DruidFilterBuilder filterBuilder;
 
+    private Optional<OrderByColumn> dateTimeSort;
```

Comment:
Since it is not a constructor parameter and no initial initialization, we don't think we can make it as final.